controller/lib/core/sensors.cpp

```diff
@@ -109,13 +98,22 @@ Sensors::Sensors() {
 //
 // @TODO: Add alarms if sensor value is out of expected range?
 Pressure Sensors::ReadPressureSensor(Sensor s) {
-  int sum = 0;
+  float sum = 0;
   for (int i = 0; i < SENSOR_SAMPLES_FOR_READ; i++) {
-    sum += Hal.analogRead(PinFor(s));
+    sum += Hal.analogRead(PinFor(s)).volts();
   }
-  // Sensitivity of all pressure sensors is 1 V/kPa; no division needed.
-  return kPa(ADC_LSB * (static_cast<float>(sum) / SENSOR_SAMPLES_FOR_READ -
-                        sensors_zero_vals_[s]));
+
+  // The pressure sensors output 1-5V, and each additional 1V of output
+  // corresponds to an additional 1kPa of pressure difference.
+  // https://www.nxp.com/docs/en/data-sheet/MPXV5004G.pdf.
+  //
+  // The pressure sensor is scaled to 0-3.3V, which is the range captured by
+  // our ADC.  Therefore, if we multiply the received voltage by 5/3.3, we get
+  // a pressure in kPa.
```

Comment:
Note that in a perfect world the sensor would output 1V for zero differential pressure, so we would see 0.66V at zero pressure after that's scaled down.
Right now we're doing a quick calibration at startup which will presumably capture that offset, but we could hard code the 0.66V offset in and just calibrate for any imperfections in the sensor.  That way if we couldn't calibrate for some reason we would still be close to the right value.
This isn't really a comment on the change you made in this PR, just an observation now that I'm looking more closely at this code.